### PR TITLE
Feature/support ctlv2 #2

### PR DESCRIPTION
**Pull request by mklooss**
Based on: https://github.com/john30/ebusd-configuration/pull/266

with Line fix

### CODE REVIEW

File: ebusd-2.1.x/de/vaillant/15.ctlv2.csv [hunk 1]
```diff
@@ -0,0 +1,385 @@
+# type (r[1-9]wu),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment
+#,,CTLV2, sensoCOMFORT VRC 720/2 (SW 0514),,,,,,,,,,
+# ##### Generell #####,,,,,,,,,,,,,
+*r,,,,,,B524,02000000,,,IGN:4,,,
+*w,,,,,,B524,02010000,,,,,,
+r;w,,ContinuosHeating,Durchgehendes Heizen,,,,0200,,,tempv,,,"-26=off, when the outside temperature falls below this threshold temperature the continuous heating function is started (off <=> function is disabled)"
```
Comment (john30) [inline ebusd-2.1.x/de/vaillant/15.ctlv2.csv:6]: spelling seems incorrect and is different from the line in the en file. should rather be continuousHeating for both

**Review by john30 — CHANGES_REQUESTED** [review]

**Comment by mklooss** [comment]
already in @janphkre repo

**Comment by AlfredJodokusQuack** [comment]
Hi guys,
Sorry for replying to this closed thread. I am still new to github...

Does this issue being closed mean that it is not going to be merged? Because I think this fix would help me with a problem. If it is not going to be merged, can somebody please tell me in which folder to put these files manually?

Thank you!

**Comment by mklooss** [comment]
@AlfredJodokusQuack see https://github.com/john30/ebusd-configuration/pull/266

**Pull request closed without merging**